src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,278 @@ redirects:
-2. When the condition is complete, you'll set up a workflow, which controls how alert violations are turned into notifications. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts and AI > Workflows**. For tips on using that, see the [workflow docs](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows) and [this post on best practices for workflows](https://discuss.newrelic.com/t/relic-solution-patterns-for-implementing-alerts-workflows/188605).
-3. Set up one or more destinations (the places notifications are sent). From **Alerts and AI**, click **Destinations**. For tips, see the [destination docs](/docs/alerts-applied-intelligence/notifications/destinations).
+<CollapserGroup>
+  <Collapser
+    id="select-your-query"
+    title="Select your query"
+  >
+  First, we'll go to the chart labeled **Web transactions time** and click **create an alert condition**. 
 
-Ideas for next steps: 
-
-* If you're confused about the above procedure, keep reading for some tips on creating a NRQL alert condition.
-* Read about [some recommended APM alerts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/condition-recommendations).
-
-## Create a NRQL alert condition [#nrql-alert] 
-
-You can write your own [NRQL query](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) and use that as the basis for a NRQL alert condition. The following will walk you through how to create a condition. 
+<img
+  title="Alert condition from a chart"
+  alt="A screenshot depicting how to create an alert condition from a chart"
+  src={alertsCreateanAlertConditionFromaChart}
+/>
 
-### Step 1: Write your query [#step1-nrql]
+<img
+  title="Select your query"
+  alt="A screenshot demonstrating how to choose your query"
+  src={alertsSelectYourQuery}
+/>
+</Collapser>
 
-You can use a NRQL query to return data about how your environment is performing. Then, you can create a condition from that query.
+  <Collapser
+    id="name-your-condition"
+    title="Name your condition"
+  >
+    It's important to give your alert condition a specific name. Let's say you name this condition **response time** and then you create another condition for a completely different application and label that condition **response time** as well, then you won't be able to distinguish which condition is for which application. So, make sure to give your condition a specific and unique name. In this case, we'd name this condition **Response time: Example-app**.
```

Comment:
I think the second sentence is a little long. How about a period after "as well" and then starting the next one with "If this occurs," or something similar?

Reply:
done!